contrib/win32/win32compat/shell-host.c

```diff
@@ -1105,18 +1105,13 @@ start_with_pty(wchar_t *command)
 {
 	STARTUPINFO si;
 	PROCESS_INFORMATION pi;
-	wchar_t *cmd = (wchar_t *)malloc(sizeof(wchar_t) * MAX_CMD_LEN);
+	wchar_t cmd[MAX_CMD_LEN] = {L'\0',};
```

Comment:
I made this change to fix warning message.. having this as non-static memory will not cause any issue?

Reply:
don't see any warning